### ISSUE TITLE
virtctl: --identity-file is not applied when setting --local-ssh=true

### ISSUE THREAD

**Issue by 0xFelix** [kubevirt/kubevirt#7071]
**Is this a BUG REPORT or FEATURE REQUEST?**:

> Uncomment only one, leave it on its own line:
>
/kind bug
> /kind enhancement

**What happened**:

I wanted to use virtctl to ssh into a VM. I specified a private key with `--identity-file` and set `--local-ssh=true`.

The executed ssh command was:

`/usr/bin/ssh -o ProxyCommand=/home/felix/.krew/bin/kubectl-virt port-forward --stdio=true vmi/rhel8-rapid-alligator.vmtest 22 cloud-user@vmi/rhel8-rapid-alligator.vmtest`

**What you expected to happen**:

The executed ssh command should be:

`/usr/bin/ssh -o ProxyCommand=/home/felix/.krew/bin/kubectl-virt port-forward --stdio=true vmi/rhel8-rapid-alligator.vmtest 22 -i /path/to/privatekey cloud-user@vmi/rhel8-rapid-alligator.vmtest`

**How to reproduce it (as minimally and precisely as possible)**:

Run `virtctl ssh user@vm --identity-file=/path/to/privatekey --local-ssh=true`

**Anything else we need to know?**:

**Environment**:
- KubeVirt version (use `virtctl version`):
`Client Version: version.Info{GitVersion:"v0.49.0", GitCommit:"f8160a0b5eac2acafab74fd7916bfcaf0c39a6a3", GitTreeState:"clean", BuildDate:"2022-01-11T17:37:26Z", GoVersion:"go1.16.6", Compiler:"gc", Platform:"linux/amd64"}
Server Version: version.Info{GitVersion:"v0.44.3-26-g08a0ea47c", GitCommit:"08a0ea47c8e883629fc6241116c1a4db4284212a", GitTreeState:"clean", BuildDate:"2021-11-26T10:05:29Z", GoVersion:"go1.16.6", Compiler:"gc", Platform:"linux/amd64"}`
- Kubernetes version (use `kubectl version`):
- VM or VMI specifications:
- Cloud provider or hardware configuration:
- OS (e.g. from /etc/os-release): Fedora 35 / RHEL 8.5
- Kernel (e.g. `uname -a`):
- Install tools:
- Others:

**Comment by mor120** [comment]
Hey, I can take this.
If It is okay, I would need a little bit of guidance.

It looks to me like when passing `--local-ssh=true`, `Run` function here - https://github.com/kubevirt/kubevirt/blob/314b515473a3720dce35f5477c6dbfdc7d544e5c/pkg/virtctl/ssh/ssh.go#L108 immediately calls https://github.com/kubevirt/kubevirt/blob/314b515473a3720dce35f5477c6dbfdc7d544e5c/pkg/virtctl/ssh/wrapped.go#L15 and runs the command without adding the `IdentityFilePath` as an argument, unlike when https://github.com/kubevirt/kubevirt/blob/314b515473a3720dce35f5477c6dbfdc7d544e5c/pkg/virtctl/ssh/ssh.go#L119 is called (when `--local-ssh` is unset) and prepares also the authentication methods by calling `getAuthMethods` that calls `tryPrivateKey` which appends a callback to the list of methods.

To summarize, what needed to be done here is to create a function in `wrapped.go` that will add a reference to `IdentityFilePath` in case `IdentityFilePathProvided` is true.
The function would be similar to the following function - 
https://github.com/kubevirt/kubevirt/blob/314b515473a3720dce35f5477c6dbfdc7d544e5c/pkg/virtctl/ssh/wrapped.go#L43

Can you please verify that I get this correctly? Thanks!

**Comment by rmohr** [comment]
Is this a duplicate of https://github.com/kubevirt/kubevirt/issues/7071 ?

**Comment by 0xFelix** [comment]
@rmohr Did you mean #7072? I would say it is not a duplicate, because in this case the ssh binary of the users machine is used. A `-i /path/to/privatekey` needs to be appended to the arguments passed to ssh.
